docs/tables/alicloud_kms_secret.md

```diff
@@ -0,0 +1,55 @@
+# Table: alicloud_kms_secret
+
+Secret enables to manage secrets in a centralized manner throughout their lifecycle (creation, retrieval, updating, and deletion).
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  description,
+  arn,
+  secret_type,
+  create_time
+from
+  alicloud_kms_secret;
+```
+
+### Get the rotation info for each secret
```

Comment:
@rajeshbal65 Can you please add 2 example queries based on the following descriptions:
`List keys that do not have automatic rotation enabled`
`List keys that have not been rotated within the last 30 days`